### PR TITLE
Update kotlin-quiz.md

### PR DESCRIPTION
**Pull request by kariot**

### CODE REVIEW

File: kotlin/kotlin-quiz.md [hunk 1]
```diff
@@ -117,15 +117,15 @@ when (die) {
 }
 ```
 
-- [ ] `3,4,5,6`
+- [x] `3,4,5,6`
 - [ ] `in 3..6`
```
Comment (LeonardoJaques) [inline kotlin/kotlin-quiz.md:121]:
```suggestion
- [x ] `in 3..6`
```

**Review by LeonardoJaques — CHANGES_REQUESTED** [review]

File: kotlin/kotlin-quiz.md (hunk 1 above)
Comment (Ebazhanov) [reply kotlin/kotlin-quiz.md:121]: @LeonardoJaques we can keep both answers but if you have an explanation that is 100% true then go ahead and put it here. We can even put your reference below the @kariot answer! I hope you guys agree, if not I am ready for any other ideas!

Comment (LeonardoJaques) [reply kotlin/kotlin-quiz.md:121]: @Ebazhanov For me its okay, you really can use both.
 I use the official site as reference -> https://kotlinlang.org/docs/ranges.html

Comment (Ebazhanov) [reply kotlin/kotlin-quiz.md:121]: ok, great! Thanks for feedback @roplayer! I will merge the changes from @kariot and later push to master your suggestion with reference link.

**Comment by Ebazhanov** [comment]
@all-contributors please add @kariot for code, design

**Comment by Ebazhanov** [comment]
@all-contributors please add @LeonardoJaques for code, design

**Pull request merged**